app/docker/views/networks/queries.ts

```diff
@@ -0,0 +1,13 @@
+import { useQuery } from 'react-query';
+
+import { EnvironmentId } from '@/portainer/environments/types';
+
+import { getNetwork } from './network.service';
+import { NetworkId } from './types';
+
+export function useNetwork(networkId: NetworkId, environmentId: EnvironmentId) {
+  return useQuery(
+    networkId && ['network', networkId, 'environmentId', environmentId],
```

Comment:
see how I did it in azure. should be something like:
`["environments", environmentId, "docker", "networks", networkId]`

this will make it possible to invalidate all networks queries by calling 
`invalidateQueries(['environments', envId, 'docker', 'networks']`

Reply:
also. networkId is required, so don't need to add `networkId &&`

Reply:
updated